### DISCUSSION TITLE
Tailwind purge to exclude critical css

### DISCUSSION THREAD

**Discussion by howdu** [tailwindlabs/tailwindcss#9604] [Help] [answered]
I have a critical.css file which gets read as style block in the head E.g:
```html
<head>
<style>.relative{position:relative}</style>
</head> 
```

Is there a way of telling tailwind to exclude this from the main css file?

I kinda want to opposite of the `content` options of file paths so that it removes the duplicate css.

**Comment by wongjn** [comment, marked as answer]
For static utilities, you could disable the core plugins which generate duplications, and rewrite them yourself to exclude the duplicates. For dynamic utilities, you could redeclare the values via `theme`, as opposed to `theme.extend` to explicitly exclude any duplicates.

Another method could be to add a PostCSS plugin after Tailwind that removes duplicate rules.
